controllers/customers.js

```diff
@@ -0,0 +1,34 @@
+var Customers = require("../models/customers");
+
+var CustomersController = {
+
+  getCustomers: function (request, response, next) {
+    Customers.all(function(error, customers) {
+      if(error) {
+        var err = new Error
+        err.status = 500;
+        err.error = "Error retrieving customer list."
+        response.json(err)
+      } else {
+        response.json(customers)
+      }
+    });
+  },
+
+  // customer id, sort column, offset (p and n)
+  getCustomersSorted: function(request, response) {
```

Comment:
I've noticed on this method you didn't include next as a param but you do so in the other controller methods.  I would recommend being consistent on whether or not you include it. (Caveat: I am not super familiar with Express and there may be a totally legitimate reason you did this :) )